src/i18n/fr_FR/buildings.ts

```diff
@@ -114,7 +128,22 @@ export default {
             credits: [10_000, 50_000, ...Array(12).fill(100_000)],
             coins: [10, 15, ...Array(12).fill(20)],
         },
-        extensions: [],
+        extensions: [
+            {
+                caption: "Grande station d'ambulance",
+                credits: 1_000_000,
+                coins: 50,
+                duration: '7 jours',
+                maxExtensionsFunction: (
+                    buildingsByType: Record<number, Building[]>
+                ): number => Math.floor((buildingsByType[2]?.length ?? 0) / 10),
```

Comment:
In the case of the fire department, police and ambulance extensions, the small stations also count in the count. you have now only added the normal stations to the count here.

Reply:
ok thanks I'll send the corrections